Sources/Shared/Models/PublishJobConnection.swift

```diff
@@ -76,8 +76,8 @@ public class PublishJobBlockers: VIMModelObject {
     }()
 
     /// Current blockers that will prevent posting to YouTube. If `nil`, publishing is not blocked for this platform.
-    public lazy var youtube: PublishBlockers? = {
-        return PublishBlockers(blockers: self.youtubeBlockers)
+    public lazy var youtube: YouTubeBlockers? = {
```

Comment:
is this the right name for this variable? I don't have the full context but perhaps it could be more explicit?

Reply:
Actually nevermind I just expanded this the PR lines and noticed it is namespaced inside `PublishJobBlockers`

Reply:
Yeah, I tried to keep the naming as similar to the API as I could. But thanks for raising it.